spec/unit/provider/package/zypper_spec.rb

```diff
@@ -32,6 +32,8 @@
 
   let(:status) { double(stdout: "\n", exitstatus: 0) }
 
+  let(:source) { "/tmp/wget_1.11.4-1ubuntu1_amd64.deb" }
```

Comment:
we shouldn't be using "deb" package names in the zypper specs, since it uses rpms.